locales/it/index.yml

```diff
@@ -911,22 +911,21 @@ wallet:
     bancomat:
       title: Ricerca delle tue carte PagoBANCOMAT
       description:
-        text1: "IO cercherà tutte le carte PagoBANCOMAT associate a te.\nProseguendo accetti l'"
+        text1: "Aggiungendo PagoBANCOMAT al tuo Portafoglio, potrai aderire alle iniziative disponibili su IO.\nProseguendo accetti l'"
         text2: informativa sul trattamento dei dati personali.
         text3: "Se preferisci, puoi scegliere la banca su cui hai una carta PagoBANCOMAT attiva, "
         text4: cercando fra quelle disponibili
     bpay:
       title: Ricerca dei tuoi account BANCOMAT Pay
       description:
-        text1: "IO cercherà tutti gli account BANCOMAT Pay associati a te.\nProseguendo accetti l'"
+        text1: "Aggiungendo BANCOMAT Pay al tuo Portafoglio, potrai aderire alle iniziative disponibili su IO.\nProseguendo accetti l'"
         text2: informativa sul trattamento dei dati personali.
         text3: "Se preferisci, puoi scegliere la banca su cui hai un account BANCOMAT Pay attivo, "
         text4: cercando fra quelle disponibili
     cobadge:
       title: Ricerca delle tue carte con circuito internazionale
       description:
-        text1: "IO cercherà tutte le carte Maestro, Mastercard, V-Pay e Visa associate a te presso "
-        text2: le banche aderenti al servizio
+        text1: "Aggiungendo al tuo Portafoglio tutte le carte Maestro, Mastercard, V-Pay e Visa a te associate, potrai aderire alle iniziative disponibili su IO."
```

Comment:
@AliceAzzolini we should keep the same keys or we must integrate with a code change because the key `wallet.searchAbi.cobadge.description.text2` is used within the code.
In this case this is used for a link:
<img width="405" alt="Schermata 2021-12-06 alle 16 25 31" src="https://user-images.githubusercontent.com/26501317/144873314-bb35db06-4ac8-4fec-88a7-be48be560902.png">

Reply:
@fabriziofff ok, I'll add a specific task to remove text2 string then, which will block this issue—thanks!

Reply:
Copy changes have been carried over to https://github.com/pagopa/io-app/pull/3692